addons/io/org.openhab.io.openhabcloud/README.md

```diff
@@ -52,3 +52,4 @@ Alternatively, you can configure the settings in the file `conf/services/openhab
 #expose=
 ```
 
+Note: The exposed items will show up in myopenhab after they receive an update in thier state.
```

Comment:
Maybe you can leave out "in myopenhab" because its just one of the possible implementations.

Als thier should be spelled as thier.

Reply:
Yes, you are right